### PR TITLE
FIX mail_footer_notified_partner

### PR DESCRIPTION
**Pull request by eLBati**
we need to save the complete list of partners because
 _message_notification_recipients builds recipients
 grouped by users groups. Thus get_additional_footer would get a
 partial list of recipients

### CODE REVIEW

**Review by SimoRubi — APPROVED** [review]
Looks good

**Comment by hurrinico** [comment]
LGTM :+1:

**Comment by eLBati** [comment]
@OCA/social-maintainers please merge

**Comment by pedrobaeza** [comment]
I would like to have ACSone approval. cc @lmignon

File: mail_footer_notified_partner/models/mail_followers.py
Comment (hbrunn) [inline mail_footer_notified_partner/models/mail_followers.py]: pass keyword arguments as keyword arguments

File: mail_footer_notified_partner/models/mail_followers.py
Comment (hbrunn) [inline mail_footer_notified_partner/models/mail_followers.py]: how about 

```python
return super(
    MailNotification, self.with_context(notified_partners=self)
)._notify_by_email(
    ...
)
```

File: mail_footer_notified_partner/models/mail_followers.py
Comment (hbrunn) [inline mail_footer_notified_partner/models/mail_followers.py]: `.get('notified_partners', recipients)`

**Review by hbrunn — CHANGES_REQUESTED** [review]
people really need to start using the github review mechanism, those thumbs up in the text slip through all filters

**Comment by hbrunn** [comment]
@pedrobaeza any reason for needing this specific approval?

**Comment by pedrobaeza** [comment]
Because they were the original authors of the module and known better the code, but your review is also the same valuable. My comment was put after the merge request from @eLBati, expressing my uncomfortability for merging this directly.

File: mail_footer_notified_partner/models/mail_followers.py
Comment (eLBati) [reply mail_footer_notified_partner/models/mail_followers.py]: @hbrunn this would produce a context with ```notified_partners``` only, right? In this case we need to keep the other items of original self.env.context

File: mail_footer_notified_partner/models/mail_followers.py
Comment (eLBati) [reply mail_footer_notified_partner/models/mail_followers.py]: @hbrunn actually ```notified_partners``` key could be present but empty. In that case, ```recipients``` should be used

File: mail_footer_notified_partner/models/mail_followers.py
Comment (hbrunn) [reply mail_footer_notified_partner/models/mail_followers.py]: https://github.com/OCA/OCB/blob/10.0/odoo/models.py#L4867

File: mail_footer_notified_partner/models/mail_followers.py
Comment (hbrunn) [reply mail_footer_notified_partner/models/mail_followers.py]: so + ` or recipients`

File: mail_footer_notified_partner/models/mail_followers.py
Comment (eLBati) [reply mail_footer_notified_partner/models/mail_followers.py]: @hbrunn thanks! fixed

File: mail_footer_notified_partner/models/mail_followers.py
Comment (eLBati) [reply mail_footer_notified_partner/models/mail_followers.py]: Fixed

File: mail_footer_notified_partner/models/mail_followers.py
Comment (eLBati) [reply mail_footer_notified_partner/models/mail_followers.py]: Fixed

**Review by gurneyalex — APPROVED** [review]

**Comment by gurneyalex** [comment]
@hbrunn  update ok with you?

**Review by hbrunn — APPROVED** [review]

**Comment by hbrunn** [comment]
in case the re-review didn't generate a notification: yes

**Pull request merged**
